dev/io.openliberty.org.jetbrains.annotation/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props,~../cnf/resources/bnd/transform.props
+
+Bundle-Name: JetBrains Annotation Jakarta
+Bundle-SymbolicName: io.openliberty.org.jetbrains.annotation
+Bundle-Description: Liberty JetBrains Annotation, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  !android.net.http,\
+  javax.net,\
+  javax.net.ssl
```

Comment:
Either use `*` or remove the excluded package. Are you sure that the `javax` packages are needed here.